### ISSUE TITLE
How to change the animation duration depends on the file upload size ?

### ISSUE THREAD

**Issue by pnk0006** [kimmobrunfeldt/progressbar.js#233]
Hi, 
I'm using this plugin for uploading file progress to show the user. the file size it will come 0.1M to 2.0M. depends on the file size how to add set the progress duration and the first 50% need to fill within a 1s and remaining 50%  3s to 4s. is there any solution for this.

**Comment by EanKeen** [comment]
Have you found a way to do this?

**Comment by morenoh149** [comment]
Yeah I find this library has most things, except I need the ability to change the animation duration between progressbar loads.
